docs/developer/channels.mdx

```diff
@@ -0,0 +1,272 @@
+---
+title: Channels
+sidebar_label: Channels
+---
+
+## Introduction
+
+Since version 3.0 Saleor supports multichannel. This feature allows the shop owner for configuring many aspects of the store depending on a channel.
+
+- Your store needs at least one channel to display products
+- You can add as many channels as you need
+- Each channel have exactly one currency
+- Channels can be deactivated for customers
+
+Popular use cases:
+- If you sell internationally, you can use separate channels for different currencies or regions
+- If you want to have different prices in the custom mobile app and website, you can set up `website-channel` and `mobile-channel`
```

Comment:
Can we prevent splitting of names like `website-channel`?
![image](https://user-images.githubusercontent.com/43955230/107225753-7d122580-6a19-11eb-946a-38ef37553e47.png)

Reply:
Good idea. Your wish has been granted! :D